client/delegate-auth/.vscode/launch.json

```diff
@@ -0,0 +1,51 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug Specific Test",
+            "type": "node",
+            "request": "launch",
+            "runtimeArgs": [
+              "--inspect-brk",
+              "${workspaceRoot}/node_modules/.bin/jest",
+              "--runInBand",
+              "--no-cache",
+              "${workspaceFolder}/test/Login.test.ts",
+            ],
+            "console": "integratedTerminal",
+            "internalConsoleOptions": "neverOpen",
+            "port": 9229
+        },
+        {
+            "name": "Debug all Test",
+            "type": "node",
+            "request": "launch",
+            "runtimeArgs": [
+              "--inspect-brk",
+              "${workspaceRoot}/node_modules/.bin/jest",
+              "--runInBand",
+              "--no-cache",
+            ],
+            "console": "integratedTerminal",
+            "internalConsoleOptions": "neverOpen",
+            "port": 9229
+        },
+        {
+          "name": "Debug script",
+          "type": "node",
+          "request": "launch",
+          "runtimeExecutable": "node",
+          "runtimeArgs": ["--nolazy", "-r", "ts-node/register/transpile-only"],
+    
+          //"args": ["lib/KiltVerifier.ts"],
+          "args": ["delegate-auth-script.ts"],
+          
+          "cwd": "${workspaceRoot}",
+          "internalConsoleOptions": "openOnSessionStart",
+          "skipFiles": ["<node_internals>/**", "node_modules/**"]
+        }
+    ]
+}
```

Comment:
I don't think we need this `launch.json` file. It looks like it was copy-pasted from `passport-did-auth`

Reply:
yes indeed